scipy/io/_fortran.py

```diff
@@ -127,55 +156,103 @@ def read_record(self, dtype=None):
 
         Notes
         -----
-        If the record contains a multi-dimensional array, calling reshape or
-        resize will restructure the array to the correct size.
-        Since Fortran multidimensional arrays are stored in column-major format,
-        this may have some non-intuitive consequences. If the variable was
-        declared as 'INTEGER var(5,4)', for example, var could be read with
-        'read_record(dtype=np.integer).reshape( (4,5) )' since Python uses
-        row-major ordering of indices.
+        If the record contains a multi-dimensional array, you can specify
+        the size in the dtype. For example::
+
+            INTEGER var(5,4)
 
-        One can transpose to obtain the indices in the same order as in Fortran.
+        can be read with::
+
+            read_record('(4,5)i4')
+
+        Note that the fact that the array is laid out in Fortran column major
+        order in the file is **not** handled automatically, and you need to take
+        care about it yourself.
```

Comment:
Perhaps this comment could be re-written to more specific instructions. I assume this is essentially the inverse operation of transposing during write_records? I think it might be easier for the user to understand if the symmetry between writing and reading is made clear.